src/rgw/rgw_crypt.cc

```diff
@@ -1090,6 +1092,78 @@ int rgw_s3_prepare_encrypt(struct req_state* s,
       }
     }
 
+    /* Checking bucket attributes if SSE is enabled. Currently only supporting SSE-S3 */
+    rgw::sal::Attrs buck_attrs(s->bucket_attrs);
+    auto aiter = buck_attrs.find(RGW_ATTR_BUCKET_ENCRYPTION);
+    if (aiter != buck_attrs.end()) {
```

Comment:
On my PR in the Get call I received comments from Casey that this should be written. (https://github.com/ceph/ceph/pull/42222#discussion_r665545701)
A similar comment might come here. So, we can rewrite this as:
`auto aiter = s->bucket_attrs.find(RGW_ATTR_BUCKET_ENCRYPTION_POLICY);
  if (aiter == s->bucket_attrs.end()) {`
  Same can be done for other references if any where get attribute is done.